src/controllers/mainController.ts

```diff
@@ -1178,6 +1187,19 @@ export default class MainController implements vscode.Disposable {
 
 			await this.sanitizeConnectionProfiles();
 
+			if (e.affectsConfiguration(Constants.cmdObjectExplorerGroupBySchemaFlagName)) {
+				this._objectExplorerProvider.rootNodeConnections.forEach(async (conn) => {
+					if (this.connectionManager.isActiveConnection(conn)) {
+						const rootNodes = await this._objectExplorerProvider.getChildren();
+						rootNodes.forEach(async (rootNode: TreeNodeInfo) => {
```

Comment:
Currently doing this means that if any errors occur while refreshing the nodes they'll be lost. I would suggest at least logging the errors if they happen (which means try/catch around the awaits)